.github/workflows/CI.yml

```
@@ -27,7 +33,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       with:
-        submodules: recursive
+        submodules: true
```

I think it's best to always do `recursive` unless there's a specific reason not to. It's almost always what you want. Even if there are no recursive submodules now, it better just work if any appear later on.

```suggestion
        submodules: recursive
```